frontend/src/components/static_pages/contact.jsx

```diff
@@ -32,48 +43,18 @@ class Contact extends Component {
             <p>&nbsp;</p>
           </div>
         </div>
-        <div className="container col-sm-6 float-left mt-5">
-          <form action="/" method="post">
-            <div>
-              <label for="name" className="col-xs-4 ">
-                Name (required)
-              </label>
-              <input id="name" required="" type="text" className="col-md-10 " />
-            </div>
-            <br />
-            <div>
-              <label for="email" className="col-xs-4">
-                Email (required)
-              </label>
-              <input
-                id="email"
-                required=""
-                type="email"
-                className="col-md-10"
-              />
-            </div>
-            <br />
-            <div>
-              <label for="phone" className="col-xs-4">
-                Phone (required)
-              </label>
-              <input id="phone" type="text" className="col-md-10" />
-            </div>
-            <br />
-
-            <div>
-              <label className="col-xs-4 mt-3 mb-1 ">Message</label>
-              <textarea name="" rows="6" className="col-md-10 mb-4" />
-            </div>
-
-            <div className="col-xs-8 mb-5 ">
-              <p>
-                <button className="button" type="submit" value="Submit">
-                  <span className="buttonSpan">Submit</span>
-                </button>
-              </p>
-            </div>
-          </form>
+        <div className="main-content container col-sm-6 mt-5">
+          <iframe
+            src="https://docs.google.com/forms/d/e/1FAIpQLSe9vicPG5oh6xdZYGLMkksNXQc_6WnjY4GCOZbjcnlVzW0huA/viewform?embedded=true"
+            style={{ width: "100%", height: this.state.iframeHeight }}
+            frameborder="0"
+            marginwidth="0"
+            marginheight="0"
+            onLoad={this.loaded}
+            className="enroll-iframe"
```

Comment:
can we update this to be `contact-iframe` so it is unique

Reply:
lets add add `title='contact-iframe'` as well to get rid of the compiler warning

![image](https://user-images.githubusercontent.com/1637481/67729528-6def9b80-f9bf-11e9-880f-134b51984e35.png)

Reply:
I added it.